src/main/java/foundation/privacybydesign/bigregister/RestApi.java

```diff
@@ -85,10 +85,10 @@ public Response getAttributes(String disclosureJWT) throws KeyManagementExceptio
 
         // These attributes can be set for debugging purposes - to impersonate someone else.
         // Must only be used in a test environment!
-        //initials = "";
-        //familyName = "";
-        //gender = "";
-        //dateOfBirthString = "";
+        //String initials = "";
+        //String familyName = "";
+        //String gender = "";
+        //String dateOfBirthString = "";
```

Comment:
This is unrelated to the PR and mostly a matter of taste I guess. The previous behavior was correct as long as the lines above (line 81-84) were not commented out.